### PR TITLE
Created copy of todo list project for review

### PR DESCRIPTION
**Pull request by shubhambhoyar077**
### To Do List

- [ ] Copied complete code from the "To Do list" project and commit it as one big chunk of code in a feature branch.
- [ ] It's have following functionality created using ES6:-

- Add
- Update
- Read
- Delete.
- And clear all completed task.

- [ ] You can check [demo](https://shubhambhoyar077.github.io/to-do-list/dist/) of original project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,181 @@
...
+
+- Clone this [repository](https://github.com/shubhambhoyar077/to-do-list) in your local machine with correct branch.
+- Terminal:- **npm run build** it will create **dist** folder. In that folder you have open index.html in any browser.
+- **or** Terminal:- **npm start** it will run on port 8080.
+
+### Run tests
+
+To run tests, open index.html in any web browser.
+
+### Deployment
```
Comment (MishDM) [inline README.md:113]: Kindly suggest that its good deploy the project  for easy visualization

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        </span></h1>
+      <div class="common-padding">
+        <input class="enter-item" id="enter-item" type="text" placeholder="Add to your list...">
+        <span id='add-item' class="material-symbols-outlined">
+          subdirectory_arrow_left
+          </span>
+      </div>
+      <ul class="list-items" id="list-items">
+      </ul>
+      <div class="clear-completed"><a id="clear-completed" href="">Clear all completed</a></div>
```
Comment (MishDM) [inline dist/index.html:24]: kindly suggest The good bit of link-accessibility advice is to write good link text if not leave it #

**Review by MishDM — COMMENTED** [review]
Hi@shubhambhoyar077 You have done great Job 💯 👍 with your project,  all thing working fine 
simple comments you have to fix 😄

File: README.md (hunk 1 above)
Comment (shubhambhoyar077) [reply README.md:113]: Thanks for your review :+1: 
Please check my [changes](https://github.com/shubhambhoyar077/to-do-list-review/pull/1/commits/208589a5367e96268efc1f6cb75e7e87bd4d91bb)

**Review by QueenterJuma — APPROVED** [review]
Hi @shubhambhoyar077 ,
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉
**STATUS: APPROVED 🎉**
unnamed

**Highlights**

- [ ] -  All linters checks are green ✔️
- [ ] -  Descriptive PR title and summary 👍
- [ ] -  Code is well structured 👍

**Optional suggestions**
 To make your remote repos lightweight, I would suggest including the coverage directory in the .gitignore file since it's not needed in your remote environment.
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**